cli/dcoscli/data/help/package.txt

```diff
@@ -3,73 +3,82 @@ Install and manage DCOS packages
 Usage:
     dcos package --config-schema
     dcos package --info
+    dcos package create [--output-directory=<output-directory>]
```

Comment:
@BenWhitehead and I talked offline. This PR only implements `create`. There will be two other PRs that implement `export` and `import`.